src/Sulu/Component/SmartContent/Orm/BaseDataProvider.php

```diff
@@ -151,21 +160,21 @@ function ($item) use ($locale) {
     }
 
     /**
-     * Decorates result as data item.
+     * Creates serialization context. Can be used to add own groups.
      *
-     * @param array $data
-     *
-     * @return array
+     * @return SerializationContext
      */
-    abstract protected function decorateDataItems(array $data);
+    protected function getSerializationContext()
+    {
+        return SerializationContext::create();
```

Comment:
Do you always serialize all the fields? I guess you made this method protected to override the context, but I can't see it used in the accounts and contacts, do you just use the default group for that there?